### PR TITLE
feat(feedback): Alignment and cleanup or Feedback Details Header UI

### LINKED ISSUE #61505: [User Feedback] feedback details should show the date when the feedback was received (relative date?)

### PR DESCRIPTION
**Pull request by ryan953**
Basically the changes are:
- alignment “better” imo
- added timestamp
- made a button Danger red
- consistent button trigger for "Copy Username" and "Copy Short ID" dropdowns

Notes:
- We're constrained a little bit still with the placement of the "assign" dropdown, see https://github.com/getsentry/sentry/pull/62723
- I do still have an item on the list for 'make issue-tracker buttons icons only'. That's coming, but there's some data-fetching stuff related that's going to happen first.

| Before | After |
| --- | --- |
| before | after

| Desc | Img |
| --- | --- |
| Date Tooltip shows Resolved time | after - tooltip
| Actions wrap down | after - wrap actions
| Actions & Timestmp wrap down | after - wrap all
| User dropdown | after - hidden dropdown - user
| Issue dropdown | after - hidden dropdown - issue id

Fixes https://github.com/getsentry/sentry/issues/61505

### CODE REVIEW

File: static/app/components/feedback/feedbackItem/feedbackTimestampsTooltip.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    activity => activity.type === 'set_resolved'
+  );
+  const dateResolved = resolvedActivity ? moment(resolvedActivity.dateCreated) : null;
+
+  return (
+    <DescriptionList>
+      <dt>{t('Created')}</dt>
+      <dd>
+        {dateFirstSeen ? (
+          <Fragment>
```
Comment (michellewzhang) [inline static/app/components/feedback/feedbackItem/feedbackTimestampsTooltip.tsx:30]: extra `Fragment`?

File: static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx [hunk 2]
```diff
@@ -1,42 +1,109 @@
...
+          position="bottom"
+          size="xs"
+          items={[
+            {
+              key: 'copy-url',
+              label: t('Copy Email Address'),
+              onAction: handleCopyEmail,
+            },
+            {
+              key: 'copy-short-id',
```
Comment (michellewzhang) [inline static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx:90]:
```suggestion
              key: 'copy-name',
```

**Review by michellewzhang — APPROVED** [review]

File: static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx (hunk 2 above)
Comment (ryan953) [reply static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx:90]: copy paste!

nice spotting :)

File: static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx [hunk 3]
```diff
@@ -1,42 +1,109 @@
...
+            icon: <IconChevron direction="down" size="xs" />,
+            size: 'zero',
+            borderless: true,
+            showChevron: false,
+          }}
+          position="bottom"
+          size="xs"
+          items={[
+            {
+              key: 'copy-url',
```
Comment (ryan953) [inline static/app/components/feedback/feedbackItem/feedbackItemUsername.tsx:85]:
```suggestion
              key: 'copy-email',
```

**Pull request merged**